Modul_6_Dzhumelya/js/index.js

```diff
@@ -0,0 +1,90 @@
+// Вправа №1 Копія масива через метод concat()
+
+const arr = ['one', 10, 'name', 5, 'Nastya', 9];
+
+function arrayCopy () {
```

Comment:
Малося на увазі, що функція повинна приймати масив як вхідний параметр.
```suggestion
function arrayCopy (array) {
```